app/core/interfaces/checkboxes/interface.js

```diff
@@ -80,10 +79,37 @@ define([
         };
       });
 
-      var customArray = {
-        key: 'custom',
-        value: values[values.length - 2]
-      };
+      function isLastValueCustom() {
+
+        //this function looks for keys of selected options,
+        // then compares these keys against the values list
+        // if anything matches, there is no custom value.
+
+        //reduces the options array to an array of selectex keys with either "undefined" or the key
+        let selectedOptions = optionsArray.map(function (option){ if (option.selected == true) { return option.key } })
```

Comment:
Whoa the future (present day) is here! I think this may be the first `let` in the Directus core code.

Reply:
Oh, we can use lets? Party on! :)

Reply:
@WellingGuzman We're allowing ES6 now?!!

Reply:
I mean that as a good thing of course 🙂

Reply:
Oh sorry, but we aren't there yet @sethjgore. Still not using ES6 yet. it was odd to see it :) I forgot to mention changing to `var`.

Again we can still talk about this, although `let` and `const` if it's well supported I'm all aboard.

Reply:
@WellingGuzman By "fully" switching to ES6, we will lose support for IE (unless we throw a transpiler like Babel in the mix). That being said, I think allowing the use of ES6 can immensely clean up the codebase in some places. Shall we open this as a 'discussion' issue?

Reply:
It's always open to discussion.